app/routes/events/components/EventEditor/EventEditor.css

```
@@ -164,3 +172,31 @@
   color: var(--secondary-font-color);
   text-align: center;
 }
+
+.groupSuggestionWrapper {
+  margin-bottom: var(--spacing-sm);
+
+  > .groupSuggestion {
+    cursor: pointer;
+    font-size: var(--font-size-xs);
+
+    &.selected {
+      background-color: var(--color-green-6);
+    }
+
+    &.implicit {
+      cursor: not-allowed;
+      background-color: var(--color-green-3);
+
+      @media (prefers-color-scheme: dark) {
```

```suggestion
      [data-theme='dark'] {
```
`prefers-color-scheme: dark` is still true in light-mode when system/browser prefers dark

---

Sweet, that explains why it wasn't working(: ended up removing it, but will add the data-theme stuff if the styling should require it